tests/commands/run_test.py

```diff
@@ -1217,3 +1222,36 @@ def test_pre_commit_env_variable_set(cap_out, store, repo_with_passing_hook):
         cap_out, store, repo_with_passing_hook, args, environ,
     )
     assert environ['PRE_COMMIT'] == '1'
+
+
+def test_pre_push_fails_if_staged_files(
+        cap_out, store, repo_with_passing_hook,
+):
```

Comment:
this test is probably sufficient -- then you don't need the additional one above